### PR TITLE
Fixed various errors

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 1]
```diff
@@ -10,20 +10,26 @@ helpdocs_is_published: true
...
 ## Overview
 
 With the increase in release velocity, the risk to code quality also increases. Using Harness Error Tracking you can:
 
-* Quickly identify and address any critical issues during each release with real-time notifications sent to email and Slack channels - ensuring the right people can make the right decisions fast.
-* Deepen your understanding of your Java applications in all environments, from production to development, testing, and staging, through code-level visibility.
-* Automate issue root-cause analysis at runtime, eliminating the need manual log analysis to identify critical errors.
-* Quickly and accurately identify and fix detected issues with Automated Root Cause Analysis (ARC) screen, leveraging source code, variable values, environment state (of the host/container) and DEBUG-level logs for full context.
-* Integrate Automated Root Cause Analysis (ARC) screen with existing tools including APMs, log analyzers, and issue-tracking software.
+* Quickly identify and address any critical issues during each release with real-time notifications sent to email and Slack channels - ensuring that the right people can make the right decisions fast.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:26]: Do we have to mention the right people?

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 2]
```diff
@@ -61,49 +66,51 @@ To create Monitored Service and define a service and environment for monitoring
...
 
-            * **Name**: Enter a name for the environment. For example, 
+            * **Name**: Enter a name for the environment.
             * **Description (Optional)**
             * **Tag (Optional)**
-            * **Environment Type**: The available options are Production and Non-Production.
+            * **Environment Type**: Select an environment type. The available options are **Production** and **Non-Production**.
 
-    Harness automatically fills in the **Monitored Service Name** field by combining the **Service** and **Environment** names.
+    Harness automatically fills the **Monitored Service Name** field by combining the **Service** and **Environment** names.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:87]: (Just a suggestion) The **Monitored Service Name** field is auto-populated by combining .....

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 3]
```diff
@@ -61,49 +66,51 @@ To create Monitored Service and define a service and environment for monitoring
...
 
 4. Select **Save**.
  
-   The newly created Monitored Service is added to the list of Monitored Services.  
+   The newly created monitored service is added to the Monitored Services list.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:91]: You can view the new monitored service in the **Monitored Services** list.
(I'm assuming users can view this list and that "Monitored Services" is a UI element.)

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 4]
```diff
@@ -112,183 +119,185 @@ To create Monitored Service and define a service and environment for monitoring
...
+* Log
+* Environment
 
 ![The Automated Root Cause (ARC) screen](./static/et-quickstart-arc-screen.png)
 
 
 ### Code tab
 
-Information displayed in the Code tab is divided into following categories:
+Information displayed in the **Code** tab is divided into following categories:
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:228]: the following

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 5]
```diff
@@ -112,183 +119,185 @@ To create Monitored Service and define a service and environment for monitoring
...
+Error Tracking captures snapshots when events, application errors (exceptions), and logs (warnings and errors) occur according to a defined algorithm.
 
 An Error Tracking snapshot contains valuable information about events in the monitored application. This includes:
 
-* Date and time of the snapshot
-* The server and application where the event occurred
-* The deployment where the event was captured
-* The full Call Stack
-* The Source Code view.
+* Date and time of the snapshot.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:250]: Do we need to add a period?

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 6]
```diff
@@ -112,183 +119,185 @@ To create Monitored Service and define a service and environment for monitoring
...
+Error Tracking enables tracing of the code and variable state associated with the event all the way back to the initiation point, where the parameters were passed. If the event involves calls across multiple machines, ARC displays a unified call stack. 
 
-To see the source code, select the method in the call stack.
+Select a method in the call stack to see its source code.
 
-The Call Stack displays the chain of methods within the environment leading up to the event. The first method in the line is the last method on a non 3rd Party code within your application.
+The call stack displays the chain of methods within the environment leading up to the event. The first method in the line is the last method on a non-third party code within your application.
 
-When an exception is caught and re-thrown once or multiple times within the thread, Related Errors drop-down displays the error analysis. This feature is available only when such exceptions exist.
+When an exception is caught and re-thrown once or multiple times within the thread, the **Related Errors** drop-down displays the error analysis. This feature is available only when such exceptions exist.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:272]: dropdown

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md [hunk 7]
```diff
@@ -300,13 +309,13 @@ In the Log tab, the error or exception lines are displayed first, followed by th
...
 :::note
-Log4j2 allows you to create custom log levels and names, however, Error Tracking reports only on log levels that are equal to or less than 300 (<span>&#8804;</span> 300).
+Log4j2 allows you to create custom log levels and names. However, Error Tracking reports only on log levels that are less than or equal to 300.
 :::
 
 
 #### Missing log statements
 
-The table bellow describes the various missing log statement you may come across in the Log tab.
+The table bellow describes the various missing log statements you may come across in the **Log** tab.
```
Comment (charanya-j) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:318]: typo
below

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 1 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:26]: Error Tracking is a continuous reliability solution in the Harness [Service Reliability Management (SRM)](https://developer.harness.io/docs/service-reliability-management/howtos-service-reliability-management/service-reliability-management-basics#sort=relevancy&f:@commonsource=%5BNextGen%20Docs%5D) module that enables developers to identify, triage, and resolve errors in applications. This helps in implementing rapid code changes by ensuring that the code is always in a deployable state.


## Overview

With the increase in release velocity, the risk to code quality also increases. Using Harness Error Tracking, as a developer, you can:

* Quickly identify and address any critical issues during each release with real-time notifications sent to email and Slack channels - ensuring that the right people can make the right decisions fast.
* Deepen understanding of your Java applications in all environments, from production to development, testing, and staging, through code level visibility.
* Automate issue root cause analysis at runtime, eliminating the need for manual log analysis to identify critical errors.
* Quickly and accurately identify and fix detected issues with Automated Root Cause Analysis (ARC) screen, leveraging source code, variable values, environment state of the host/container, and DEBUG level logs to get the full context.
* Integrate the ARC screen with existing tools including APMs, log analyzers, and issue tracking software.

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 2 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:87]: Corrected - Harness autopopulates the **Monitored Service Name** field by combining the **Service** and **Environment** names.

Note: I have considered auto-populate as one word based on the discussion here:
https://english.stackexchange.com/questions/7657/use-of-hypens-with-auto-autopopulate-auto-populate-or-auto-populate

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 3 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:91]: Done

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 5 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:250]: Removed

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 7 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:318]: Corrected

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md (hunk 6 above)
Comment (SudheendraKatte) [reply docs/service-reliability-management/use-service-reliability-management/error-tracking-category/error-tracking-in-srm.md:272]: Corrected

**Review by charanya-j — APPROVED** [review]

**Pull request merged**